sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/_client.py

```diff
@@ -0,0 +1,111 @@
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import Any, TYPE_CHECKING
+
+from azure.core import PipelineClient
+from azure.core.credentials import AzureKeyCredential
+from azure.core.rest import HttpRequest, HttpResponse
+
+from ._configuration import AnomalyDetectorClientConfiguration
+from ._operations import AnomalyDetectorClientOperationsMixin
+from ._serialization import Deserializer, Serializer
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Dict
+
+
+class AnomalyDetectorClient(AnomalyDetectorClientOperationsMixin):  # pylint: disable=client-accepts-api-version-keyword
+    """The Anomaly Detector API detects anomalies automatically in time series data. It supports two
+    kinds of mode, one is for stateless using, another is for stateful using. In stateless mode,
+    there are three functionalities. Entire Detect is for detecting the whole series with model
+    trained by the time series, Last Detect is detecting last point with model trained by points
+    before. ChangePoint Detect is for detecting trend changes in time series. In stateful mode,
+    user can store time series, the stored time series will be used for detection anomalies. Under
+    this mode, user can still use the above three functionalities by only giving a time range
+    without preparing time series in client side. Besides the above three functionalities, stateful
+    model also provide group based detection and labeling service. By leveraging labeling service
+    user can provide labels for each detection result, these labels will be used for retuning or
+    regenerating detection models. Inconsistency detection is a kind of group based detection, this
+    detection will find inconsistency ones in a set of time series. By using anomaly detector
+    service, business customers can discover incidents and establish a logic flow for root cause
+    analysis.
+
+    :param endpoint: Supported Cognitive Services endpoints (protocol and hostname, for example:
+     https://westus2.api.cognitive.microsoft.com). Required.
+    :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.AzureKeyCredential
+    :param api_version: Anomaly Detector API version (for example, v1.1). "v1.1" Default value is
+     "v1.1".
+    :type api_version: str
+    :keyword endpoint: Service URL. Required. Default value is "".
+    :paramtype endpoint: str
+    """
+
+    def __init__(
+        self,
+        endpoint: str,
+        credential: AzureKeyCredential,
+        api_version: str = "v1.1",
+        *,
+        endpoint: str = "",
+        **kwargs: Any
+    ) -> None:
```

Comment:
two endpoint is caused by lack of `x-ms-parameterized-host` in https://github.com/Azure/azure-rest-api-specs/blob/feature/cognitiveservices/anomalydetector/mvad/specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json